components/camel-chatscript/src/main/java/org/chatscript/Bot/ChatScriptBot.java

```diff
@@ -0,0 +1,76 @@
+package org.chatscript.Bot;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.Socket;
+import java.util.HashMap;
+
+import org.apache.camel.component.ChatScriptProducer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ChatScriptBot {
+
+	String host=null;
```

Comment:
did not check, but i can see indent differences.
could you please run,

mvn clean install -Psourcecheck

Reply:
Hi, I see some issues being reported by sourcecheck. I will fix and commit the same.

Reply:
I had to re-organized the package structure, Fixed all the checkstyle issues,

[INFO] --- maven-checkstyle-plugin:3.0.0:checkstyle (default-cli) @ camel-chatscript ---
[INFO] Starting audit...
Audit done.
[INFO]
[INFO] --- maven-remote-resources-plugin:1.5:process (process-resource-bundles) @ camel-chatscript ---
[INFO]